prime-angular-frontend/src/app/modules/site-registration/shared/guards/site.guard.ts

```diff
@@ -40,9 +38,71 @@ export class SiteGuard extends BaseGuard {
           // Store the site for access throughout creation and updating of a
           // site, which will allows provide the most up-to-date site
           this.siteService.site = site;
-          // TODO always resolve until routes are lock down
-          return true;
+
+          return this.routeDestination(routePath, site);
         })
       );
   }
+
+  /**
+   * @description
+   * Determine the route destination based on the site status.
+   */
+  private routeDestination(routePath: string, site: Site) {
+    if (site) {
+      return (site.submittedDate)
+        ? this.manageSubmittedSiteRouting(routePath, site)
+        : true;
+    }
+
+    // Otherwise, prevent the route from resolving
+    return false;
+  }
+
+  private manageSubmittedSiteRouting(routePath: string, site: Site) {
+    return this.manageRouting(routePath, SiteRoutes.SITE_MANAGEMENT, site);
+  }
+
+  private manageRouting(routePath: string, defaultRoute: string, site: Site): boolean {
+    let childRoute = routePath.includes('remote-users')
```

Comment:
Also, I might have commented why we're checking for inclusion of the `remote-users` even though it is in `editRegistrationRouteOrder` since another developer might not go an scrutinize the routing module and make the connection.  Alternatively, you could skip this check entirely and add `remote-user` to `editRegistrationRouteOrder ` and rename it to `editRegistrationRouteAccess` since the doesn't control order after the registration has been submitted.